src/data_info/routes/schemas/stats.clj

```diff
@@ -13,4 +13,9 @@
 
 (s/defschema FilteredStatQueryParams
   (merge StandardUserQueryParams
-         stats-schema/FilteredStatQueryParams))
+         stats-schema/FilteredStatQueryParams
+         {(s/optional-key :ignore-missing)
+          (describe Boolean "If set to true, missing paths or data ids will be ignored.")
+
+          (s/optional-key :ignore-inaccessible)
+          (describe Boolean "If set to true, inaccessible paths or data ids will be ignored.")}))
```

Comment:
I was originally going to use `data IDs` or simply `IDs`, but the description of the request body uses `data ids`. I thought it was best to remain consistent.

Reply:
Heh, this was very possibly me being lazy, but it's probably still reasonable to keep it consistent. Internal documentation anyway, I think.

Reply:
Yep, it's mostly internal, and I understood what it meant. I was mostly preemptively addressing potential review comments. 😄